oga/src/components/MapSearchBox/MapSearchBox.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Summary.
+ * MapSearchBox for Maps, using google maps API.
+ * Referred from google-map-react examples
+ *
+ * Description.
+ * MapSearchBox component that provides recommendations near the map location.
+ * @author taehioum
+ * @since  2019-10-18
+ */
+ 
+/** jshint {inline configuration here} */
+import React, { Component } from 'react';
+import './MapSearchBox.css';
+
+class MapSearchBox extends Component {
+  constructor(props) {
+    super(props);
+    this.clearSearchBox = this.clearSearchBox.bind(this);
```

Comment:
How about changing this.clearSearchBox.bind(this) to arrow function like this.clearSearchBox => ();?

Reply:
I will look in to that.